cashu/mint/app.py

```diff
@@ -35,7 +34,8 @@ def __init__(self):
                 self.padding = 0
                 self.minimal_fmt: str = "<green>{time:YYYY-MM-DD HH:mm:ss.SS}</green> | <level>{level}</level> | <level>{message}</level>\n"
                 if settings.debug:
-                    self.fmt: str = "<green>{time:YYYY-MM-DD HH:mm:ss.SS}</green> | <level>{level: <4}</level> | <cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> | <level>{message}</level>\n"
+                    self.fmt: str = "<green>{time:YYYY-MM-DD HH:mm:ss.SS}</green> | <level>{level: <4}</level> | <cyan>{name}</cyan>:<cyan>{function}"
+                    "</cyan>:<cyan>{line}</cyan> | <level>{message}</level>\n"
```

Comment:
Not sure what `black` thinks but multi-line string concat feels clearer if it's within brackets

Reply:
Yeah I think as with the tests, this works syntactivally but needs to be in brackets for the strings to concatenate together